### ISSUE TITLE
Extract databases in go schema

### ISSUE THREAD

**Issue by wesbillman** [TBD54566975/ftl#1120]

**Comment by wesbillman** [comment]
I see this in schema now
```
// This is the echo module.
module echo {
  config default String

  config secret String

  database postgres echo

  // An echo request.
  data EchoRequest {
    name String? +alias json "name"
  }

  data EchoResponse {
    message String +alias json "message"
  }

  // Echo returns a greeting with the current time.
  verb echo(echo.EchoRequest) echo.EchoResponse
    +calls time.time
}
```

**Comment by worstell** [comment]
fixed with #1320
